api/tests/opentrons/protocol_api/test_protocol_context.py

```diff
@@ -472,7 +472,7 @@ def test_load_module_with_configuration(subject: ProtocolContext) -> None:
         )
 
 
-@pytest.mark.parametrize("api_version", [APIVersion(2, 14)])
+@pytest.mark.parametrize("api_version", [APIVersion(2, 15)])
```

Comment:
This line is my bad, got confused with the versioning. Because this should raise an error, we want to use version 2.14 or earlier for this. So your previous line was correct.

```suggestion
@pytest.mark.parametrize("api_version", [APIVersion(2, 14)])
```

Reply:
This test should be failing in the suite/CI because we are parametrizing the version as 2.15. Is that not the case?